bin/init

```diff
@@ -2,7 +2,6 @@
 
 # This script will setup the database and then start the rails application
 
-set -x
 set -e
 
 : ${VELUM_PORT:=80}
```

Comment:
Yes! Thanks! (We are never falling back to the default, but with this changes the default should be `443`).